package.json

```diff
@@ -93,5 +93,9 @@
   },
   "engines": {
     "node": "8.* || >= 10.*"
+  },
+  "volta": {
```

Comment:
sir, what is this? :P

Reply:
@locks I'm assuming you're just teasing? If not ... https://volta.sh